encoding/masking-limiting-etc.md

```diff
@@ -400,12 +401,12 @@ integer and float formats,
 so for more complex filtering
 float is recommended whenever possible.
 In 8 bit integer format where neutral luminance (gray) is 128,
-the function is `clip1 - clip2 + 128` for MakeDiff
-and `clip1 + clip2 - 128` for MergeDiff,
+the function is $$\big( \mathrm{clip\, a} - \mathrm{clip\, b} + 128 \big)$$ for MakeDiff
```

Comment:
Why do you use `\big` here?

Reply:
#### big
![image](https://user-images.githubusercontent.com/10972040/66611922-f41d7c80-eb8d-11e9-9eab-e0b6f89ec774.png)
#### normal parens
![image](https://user-images.githubusercontent.com/10972040/66611933-01d30200-eb8e-11e9-8c72-e8f9697a7bd3.png)

It's subtle but I think it helps to separate the formulas from the text better. Originally, the in-line would've had a dark background to separate it.

#### original
![image](https://user-images.githubusercontent.com/10972040/66612009-4b235180-eb8e-11e9-9b07-6ca831e92ed2.png)

Reply:
Well, that probably should have read "Why do you use `\big(`?", but I see that it's probably to make the formula stand out more from the text.
Can we target them with a CSS selector instead?

Reply:
Yea, we actually can. What did you have in mind?

Reply:
I could add a background to in-lines like they originally had

Reply:
Yeah, I think something for the background would be nice. A semi-transparent black/white, depending on the theme.